app/views/requests_confirmation_mailer/confirmation_email.html.erb

```diff
@@ -2,15 +2,14 @@
 
 <p>This email confirms that <%= @organization.name %> has received a request<%= " submitted by #{@requester_user_name}" if @requester_user_name.present? %> for:</p>
 <ul>
-  <% @request_items.each do |item| %>
-    <li><%= item['name'] %> -
-    <% if Flipper.enabled?(:enable_packs) && item['unit'] %>
-    <%= pluralize(item['quantity'] || item['person_count'], item['unit']) %>
+  <% @item_requests.each do |item_request| %>
+    <li><%= item_request.item&.name || item_request.name %> -
```

Comment:
Should we have a quick method that does this? It's repeated in several places now.